README.md

```diff
@@ -55,6 +64,9 @@ Write your code here...
 
 ```ruby
 # code here
+town[:guests] = town[:residents].fetch(1)
+town[:residents].delete("Belle")
+
 ```
```

Comment:
Your first line of code creates a new key called `guests` and sets as the value "Belle".  The existing `guests` array is still there, however, and still empty. 

A solution:

```rb
town[:castle][:guests] << "Belle"
```